flink-streaming-java/src/test/java/org/apache/flink/streaming/api/functions/sink/TwoPhaseCommitSinkFunctionTest.java

```diff
@@ -194,29 +348,4 @@ public String toString() {
 			return String.format("FileTransaction[%s]", tmpFile.getName());
 		}
 	}
-
-	private static class TestContext implements AutoCloseable {
```

Comment:
Why did you remove this encapsulation?

Reply:
I am using 
```
	@Rule
	public TemporaryFolder folder = new TemporaryFolder();
```
to create test folders are files which are cleaned up automatically. Since the rule must be a public field in the test, it would have required more work to keep the `TestContext`.